index.html

```diff
@@ -165,11 +165,20 @@ <h2>Current Committee</h2>
                 </ul>
               </li>
             </ul>
+
+            <p><strong>Changes Feb 27, 2018</strong></p>
+            <ul>
+              <li><a href="https://twitter.com/raquelxmoss" target="_blank">Raquel Moss</a> <strong>Stepping down as President</strong></li>
+              <li><strong>New President:</strong> <a href="https://twitter.com/nz_cub3y" target="_blank">Charles Peach</a></li>
+            </ul>
+
+            <p>Special thanks to <a href="https://twitter.com/raquelxmoss" target="_blank">Raquel Moss</a> for leading the current committee 2017-2018!</p>
           </section>
           <hr />
           <section id="past-committee">
             <h2>Past Committee Members</h2>
             <ul>
+              <li><a href="https://twitter.com/raquelxmoss" target="_blank">Raquel Moss</a> (president 2017-2018)</li>
```

Comment:
@raquelxmoss you were a general member from 2016 right?

Reply:
Yes indeed 😄

Reply:
@danielfone @raquelxmoss 🤔 whoops that was missing! Ill add you in!